### PR TITLE
edited skills of ayush bhatt

### PR DESCRIPTION
**Pull request by ayushbhatt2000**
done

### CODE REVIEW

**Comment by GangaChatrvedi** [comment]
I think link for your LinkedIn account is not correct as it shows page not found. Please correct it.

File: src/data/contributors/ayushbhatt.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "Ayush Bhatt",
-  "skills": ["Full Stack Development"],
+  "skills": ["Java","Data Structures"],
   "cause": "Self Empowerment",
   "image": "Avatar3",
   "github": "https://github.com/ayushbhatt2000",
   "twitter": "https://twitter.com/ayushbhatt2001",
-  "linkedin": "www.linkedin.com/in/ayushbhatt-"
+  "linkedin": "www.linkedin.com/in/ayushbhattin"
```
Comment (GangaChatrvedi) [inline src/data/contributors/ayushbhatt.json:8]: You should put https in url.
```
https://www.linkedin.com/in/ayushbhattin
```

**Comment by Abhishek-kumar09** [comment]
@ayushbhatt2000 ping!

**Comment by KeenWarrior** [comment]
@ayushbhatt2000 reopen when completed the changes 😃

**Pull request closed without merging**